src/datepicker/bs-daterangepicker-inline.component.ts

```diff
@@ -55,6 +55,10 @@ export class BsDaterangepickerInlineDirective implements OnInit, OnDestroy, OnCh
      * Disable specific dates
      */
     @Input() datesDisabled: Date[];
+    /**
+     * Disable specific dates
+     */
+    @Input() datesEnabled: Date[];
```

Comment:
Why this property is mandatory for this directive? Is it correct?

Reply:
for daterangepicker-inline and datepicker-inline all Inputs are mandatory.
Maybe, that was wrong from the beginning, and we should change that